certified-connectors/OpenLegacy IBM I (AS400)/apiDefinition.swagger.json

```diff
@@ -1,318 +1,477 @@
-                "x-ms-visibility": "internal",
-                "responses": {
-                    "200": {
-                        "description": "OK",
-                        "schema": {
-                            "$ref": "#/definitions/GetMethodsForContractResponse"
-                        }
-                    }
-                }
-            }
-        },
-        "/api/v1/partners/method-swagger-by-id/{methodId}": {
-            "get": {
-                "summary": "Get Method Open Api Spec",
-                "operationId": "GetMethodOpenApiSpec",
-                "description": "Get a method OpenApi specification",
-                "parameters": [
-                    {
-                        "name": "methodId",
-                        "in": "path",
-                        "required": true,
-                        "type": "string",
-                        "x-ms-url-encoding": "single",
-                        "x-ms-summary": "Method ID for which an OpenApi specification must be returned.",
-                        "description": "Method ID for which an OpenApi specification must be returned."
-                    }
-                ],
-                "x-ms-visibility": "internal",
-                "responses": {
-                    "200": {
-                        "description": "OK",
-                        "schema": {
-                            "type": "object",
-                            "properties": {
-                                "flow-input": {
-                                    "type": "object"
-                                },
-                                "flow-output": {
-                                    "type": "object"
-                                }
-                            }
-                        }
-                    }
-                }
-            }
+  "swagger": "2.0",
+  "info": {
+      "title": "OpenLegacy IBM I (AS400)",
+      "description": "OpenLegacy IBM i (AS400) connector provides bidirectional connectivity to your system’s data and business logic through APIs deployed on OpenLegacy’s cloud or any other target deployment. You can now build Business Flows with information from your legacy environment using the OpenLegacy connector. This will enable you to utilize your legacy assets as digital components, providing you a legacy integration and modernization solution from Day 1.",
+      "version": "1.0",
+      "contact": {
+          "name": "OpenLegacy Support",
+          "url": "https://www.openlegacy.com/company/support",
+          "email": "support@openlegacy.com"
+      }
+  },
+  "host": "api.ol-hub.com",
+  "basePath": "/backend",
+  "schemes": [
+      "https"
+  ],
+  "x-ms-connector-metadata": [
+      {
+          "propertyName": "Website",
+          "propertyValue": "https://www.openlegacy.com"
         },
-        "/dummy": {
-            "post": {
-                "summary": "Perform Request",
-                "operationId": "PerformRequest",
```

Comment:
@boiv The operation has been deleted. This change breaks existing flows. Since this is a new connector, we can ignore this error. but please be aware in the future for this kind of changes. We will ask for a confirmation that no users will be affected from this change.
You can resolve this conversation.
Thank you so much again.

Reply:
Thank you @Amjed-Ayoub.
Yes, the operation was renamed due to the fact, that Product team decided to introduce additional operations. And as you wrote, this is a new connector, that's why I renamed it without any worries, but in future I will keep in mind that this is breaking change